solr/core/src/test/org/apache/solr/cloud/TestStressInPlaceUpdates.java

```diff
@@ -485,7 +485,7 @@ public void run() {
                   } else {
                     fail(
                         String.format(
-                            Locale.ENGLISH, "There were more than one result: {}", response));
```

Comment:
I always liked the `{}`, but that my be the rubyist in me, where string interpolation is `"String #{mystring}"`

Reply:
Sure but its not necessary and makes this line break funny too.